harmonica/forward/point_mass.py

```diff
@@ -249,6 +257,32 @@ def kernel_g_z_cartesian(easting, northing, upward, easting_p, northing_p, upwar
     return (upward - upward_p) / distance ** 3
 
 
+@jit(nopython=True)
+def kernel_g_northing_cartesian(
+    easting, northing, upward, easting_p, northing_p, upward_p
+):
+    """
+    Kernel function for northing component of gravity gradient in Cartesian coordinates
+    """
+    distance = distance_cartesian(
+        [easting, northing, upward], [easting_p, northing_p, upward_p]
+    )
+    return (northing - northing_p) / distance ** 3
+
+
+@jit(nopython=True)
+def kernel_g_easting_cartesian(
+    easting, northing, upward, easting_p, northing_p, upward_p
+):
+    """
+    Kernel function for easting component of gravity gradient in Cartesian coordinates
+    """
+    distance = distance_cartesian(
+        [easting, northing, upward], [easting_p, northing_p, upward_p]
+    )
+    return (easting - easting_p) / distance ** 3
```

Comment:
Same for the `kernel_g_northing_cartesian`:
```suggestion
    return - (easting - easting_p) / distance ** 3
```

Reply:
ok